.travis.yml

```diff
@@ -24,10 +24,12 @@ install:
   #   /home/travis/gopath/src/github.com/harmony-one/harmony
   # https://docs.travis-ci.com/user/languages/go/#go-import-path
   - echo $TRAVIS_PULL_REQUEST_BRANCH
-  - TEST_REPO_BRANCH="master"
+    #TODO: return back to master
+  - TEST_REPO_BRANCH="feat_1.22_update"
   - git clone https://github.com/harmony-one/mcl.git $GOPATH/src/github.com/harmony-one/mcl
   - git clone https://github.com/harmony-one/bls.git $GOPATH/src/github.com/harmony-one/bls
-  - git clone --branch $TEST_REPO_BRANCH https://github.com/harmony-one/harmony-test.git $GOPATH/src/github.com/harmony-one/harmony-test
+    #TODO: return back harmony instead of personal fork
+  - git clone --branch $TEST_REPO_BRANCH https://github.com/mur-me/harmony-test.git $GOPATH/src/github.com/harmony-one/harmony-test
```

Comment:
Will we return back after PR will be merged?

Reply:
@mur-me why are we using your fork and not the harmony github account ?

Reply:
'cause I don't have a write access to the `harmony-test` repo

Reply:
ok i see, i created feature/go1.22 on harmony-one/harmony-test repo. can you make the update accordingly here. Thanks